bin/rails

```diff
@@ -1,4 +0,0 @@
-#!/usr/bin/env ruby
```

Comment:
I don't think this file is meant to be deleted. That is not what railsdiff suggests doing. Shall I re-instate it @pmanrubia? Asking you in absence of Ikenna for advice.